Dockerfile

```diff
@@ -5,7 +5,10 @@ RUN apt-get update && \
     apt-get install -y git libgflags-dev libpopt-dev \
                        libgoogle-glog-dev liblua5.1-0-dev \
                        libboost-all-dev libqt5websockets5-dev \
-                       python-is-python3 libeigen3-dev sudo
+                       python-is-python3 libeigen3-dev sudo 
+
+# multiplexers to run roscore and other infra in background
+RUN apt-get install -y tmux
```

Comment:
combine this with the previous apt command to minimize layers

Reply:
Done.